Gruntfile.js

```diff
@@ -309,12 +317,12 @@ module.exports = function(grunt) {
         },
 
         umd: {
-            dist: {
+            components: {
                 options: {
                     src: 'dist/d3fc.js',
                     objectToExport: 'fc',
                     deps: {
-                        'default': ['d3', 'cssLayout'],
+                        'default': ['d3', 'computeLayout'],
```

Comment:
Yea, my bad, I didn't test the UMD plugin version.

On Mon, 14 Sep 2015 at 21:46 Colin Eberhardt <notifications@github.com>
wrote:

> In Gruntfile.js
> <https://github.com/ScottLogic/d3fc/pull/550#discussion_r39444640>:
>
> >                  options: {
> >                      src: 'dist/d3fc.js',
> >                      objectToExport: 'fc',
> >                      deps: {
> > -                        'default': ['d3', 'cssLayout'],
> > +                        'default': ['d3', 'computeLayout'],
>
> well spotted - I guess this was wrong previously?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ScottLogic/d3fc/pull/550/files#r39444640>.
>